lib/resources/example_json/r4_examples_condition.rb

```diff
@@ -237,6 +274,17 @@ module Resources
             }
           ],
           'text': 'Health Concern'
+        },
+        {
```

Comment:
health concerns won't the classification.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/701/commits/b8983bad284d00a1972186571002ee91b6559052#diff-20bf74979e847ab2cdb40b65f16e39069096d8a3a39fefe73adeae819dc6bbdfL241